server/app/api.py

```diff
@@ -1821,7 +1763,8 @@ def get_courses(self, course, user, data):
     def get_students(self, course, user, data):
         query = models.Participant.query(
             models.Participant.course == course.key,
-            models.Participant.role == 'student')
+            models.Participant.role == 'student',
+            models.Participant.status != 'inactive')
```

Comment:
Try merging master in again? This is old code that can break ok (that was reverted).